### PR TITLE
[VEGA-4682]: Update consta/uikit and hilight table rows with Resources type.

### PR DESCRIPTION
**Pull request by ndrzhn**
## Problem statement

VEGA-4682 Нужно подсвечивать строки в таблице результатов расчета в зависимости от их типа (запасы - стандартный цвет, ресурсы - более светлый).

## Solution details

Было использовано свойство getAdditionalClassName компонента Table из новой версии библиотеки Consta.
Стенд: http://nd-4574-pwc-vega-builder.code013.org/

## Type

- [x] Feature
- [ ] Bug Fix
- [ ] Refactoring
- [ ] Test
- [ ] Other

## Associated issues

-

## Quality control

- [x] PR: Based on a correct branch
- [x] PR: Head branch is rebased on the base branch
- [ ] PR: Assignee chosen and necessary labels are set
- [ ] PR: Current form is filled out (where it makes sense)
- [x] PR: Diff checked for irrelevant changes
- [x] PR: Commit messages follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specs
- [x] PR: Checked for spelling, grammar and typos
- [x] JS: There's no errors/warnings in the browser dev console
- [x] Layout: Tested with various content amounts
- [x] Docs: All the important changes and features are described
- [ ] Tests: Configuration files are checked at test repositories
- [ ] Tests: New unit tests developed
- [ ] Tests: New E2E tests developed
- [ ] Tests: Existing unit tests passed successfully
- [ ] Tests: Existing E2E tests passed successfully
- [x] Tests: Manually tested on personal instance

## Notable statements
- [ ] Affects configuration files
- [x] Brings new packages or updates existing
- [ ] Opened follow-up tasks to resolve

### CODE REVIEW

File: src/components/TableResultRbController/TableResultRb/TableResultRb.css [hunk 1]
```diff
@@ -20,6 +20,10 @@
     color: var(--color-typo-primary);
   }
 
+  .TableCell_lightBackground {
+    background-color: rgb(60, 64, 67);
```
Comment (SergeyKatugin) [inline src/components/TableResultRbController/TableResultRb/TableResultRb.css:24]: константу бы, они лежат тут https://consta-uikit.vercel.app/?path=/docs/components-theme--playground

File: src/components/TableResultRbController/TableResultRb/TableResultRb.tsx [hunk 2]
```diff
@@ -394,6 +394,13 @@ export const TableResultRb: React.FC<Props> = ({
         borderBetweenRows
         stickyHeader
         isResizable
+        getAdditionalClassName={({ column, row, isActive }) => {
```
Comment (SergeyKatugin) [inline src/components/TableResultRbController/TableResultRb/TableResultRb.tsx:397]: типа нет?

File: src/components/TableResultRbController/TableResultRb/TableResultRb.tsx [hunk 3]
```diff
@@ -394,6 +394,13 @@ export const TableResultRb: React.FC<Props> = ({
...
+          return row.GEO_CATEGORY?.value === 'Ресурсы' &&
```
Comment (SergeyKatugin) [inline src/components/TableResultRbController/TableResultRb/TableResultRb.tsx:398]: Константу бы. И по коду можем сравнить?

File: src/components/TableResultRbController/TableResultRb/TableResultRb.tsx [hunk 4]
```diff
@@ -394,6 +394,13 @@ export const TableResultRb: React.FC<Props> = ({
...
+            !column.mergeCells &&
+            !isActive
+            ? 'TableCell_lightBackground'
+            : '';
+        }}
```
Comment (SergeyKatugin) [inline src/components/TableResultRbController/TableResultRb/TableResultRb.tsx:403]: Ну и в метод. Чтоб тут было что то вроде `getAdditionalClassName={getAdditionalClassName}`

File: src/components/TableResultRbController/TableResultRb/TableResultRb.css (hunk 1 above)
Comment (ndrzhn) [reply src/components/TableResultRbController/TableResultRb/TableResultRb.css]: исправила

File: src/components/TableResultRbController/TableResultRb/TableResultRb.tsx (hunk 2 above)
Comment (ndrzhn) [reply src/components/TableResultRbController/TableResultRb/TableResultRb.tsx]: нет

File: src/components/TableResultRbController/TableResultRb/TableResultRb.tsx (hunk 3 above)
Comment (ndrzhn) [reply src/components/TableResultRbController/TableResultRb/TableResultRb.tsx]: вынесла в enum

File: src/components/TableResultRbController/TableResultRb/TableResultRb.tsx (hunk 4 above)
Comment (ndrzhn) [reply src/components/TableResultRbController/TableResultRb/TableResultRb.tsx]: вынесла в метод

File: src/components/TableResultRbController/TableResultRb/TableResultRb.css [hunk 5]
```diff
@@ -27,4 +27,8 @@
   ._no-right-border {
     border-left: 0 !important;
   }
+
+  ._lightBackground {
```
Comment (SergeyKatugin) [inline src/components/TableResultRbController/TableResultRb/TableResultRb.css:31]: в БЭМ используется kebab-case

Comment (ndrzhn) [reply src/components/TableResultRbController/TableResultRb/TableResultRb.css]: спасибо, поменяла

**Pull request merged**